db/migrate/20160114203143_change_type_of_legacy_courses.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeTypeOfLegacyCourses < ActiveRecord::Migration
+  def up
+    Course.where('id < ?', 10000).update_all(type: 'LegacyCourse')
+  end
+
+  def down
+    Course.where('id < ?', 10000).update_all(type: 'ClassroomProgramCourse')
```

Comment:
But the constant that defines this is going away. Should I define that as a constant within the migration? Seems pointless, since it's a migration.